Assets/Scripts/UI/PauseMenu.cs

```diff
@@ -0,0 +1,47 @@
+﻿using TMPro;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace UI
+{
+    public class PauseMenu : Menu
+    {
+        [SerializeField] private GameObject pausePanel;
+
+        // Update is called once per frame
+        private void Update()
+        {
+            PauseGame();
+        }
+
+        public void SetHighScoreText()
+        {
+            var highScores = ScoreManager.GetScores();
+            var scoreTextBoxes = highScoreTextPanel.GetComponentsInChildren<TextMeshProUGUI>();
+            
+            SetHighScoreText(highScores, scoreTextBoxes);
+        }
```

Comment:
This function is duplicated in PauseMenu and MainMenuScript classes, I suggest we move it in the Menu class, avoiding the code duplication

Reply:
Oh good catch, I don't know why I did that considering I put the rest of the code in the parent class 😅